tests/integration_tests/functional/test_feat_parity.py

```diff
@@ -63,32 +36,46 @@ def inst_set_cpu_template_ext_fxt(request):
     return request.param
 
 
-def create_vm(vm_builder, cpu_template, microvm, kernel, disk):
+@pytest.fixture(name="vm")
+def vm_fxt(
+    microvm_factory,
+    inst_set_cpu_template,
+    guest_kernel,
+    rootfs_msrtools,
+    network_config,
+):
     """
     Create a VM.
     """
-    root_disk = disk.copy()
-    vm_instance = vm_builder.build(
-        kernel=kernel,
-        disks=[root_disk],
-        ssh_key=disk.ssh_key(),
-        config=microvm,
-        cpu_template=cpu_template,
-    )
-    vm = vm_instance.vm
-
+    vm = microvm_factory.build(guest_kernel, rootfs_msrtools)
+    vm.spawn()
+    vm.basic_config(vcpu_count=1, mem_size_mib=1024, cpu_template=inst_set_cpu_template)
+    _tap, _, _ = vm.ssh_network_config(network_config, "1")
+    vm.start()
     return vm
 
 
-def check_cpuid_feat_flags(
-    vm_builder, cpu_template, microvm, kernel, disk, must_be_set, must_be_unset
+@pytest.fixture(name="vm_ext")
+def vm_ext_fxt(
+    microvm_factory, inst_set_cpu_template_ext, guest_kernel, rootfs, network_config
 ):
     """
-    Check that CPUID feature flag are set and unset as expected.
+    Create a VM.
```

Comment:
Maybe worth mentioning what is the difference from the vm_fxt?
Also, having a more specific goal of the change in the commit message would help.

Reply:
Yeah as it is it looks gratuitous, but some other change made the `from conftest ...` import now work, and I just went ahead converting it altogether.

Reply:
fixed, I hope